docker/m3coordinator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# stage 1: build
+FROM golang:1.10-alpine AS builder
+LABEL maintainer="The M3DB Authors <m3db@googlegroups.com>"
+
+# Install Glide
+RUN apk add --update glide git make bash
```

Comment:
We should reuse these steps across the Dockerfiles at some point, but let's just move forward quickly for now to unblock others using the images?